website/templates/nav.mako

```diff
@@ -4,99 +4,91 @@
 
 <nav class="navbar navbar-inverse navbar-fixed-top" id="navbarScope" role="navigation">
     <div class="container">
-    <div class="navbar-header">
-      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
-        <span class="sr-only">Toggle navigation</span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-      </button>
-      <!-- ko ifnot: onSearchPage -->
-      <span class="visible-xs" data-bind="click : toggleSearch, css: searchCSS">
-          <a class="osf-xs-search pull-right" style="padding-top: 12px" >
-            <span rel="tooltip" data-placement="bottom" title="Search OSF" class="fa fa-search fa-lg fa-inverse" ></span>
-          </a>
-      </span>
-      <!-- /ko -->
-      <a class="navbar-brand hidden-sm hidden-xs" href="/"><img src="/static/img/cos-white2.png" class="osf-navbar-logo" width="27" alt="COS logo"/> Open Science Framework</a>
-      <a class="navbar-brand visible-sm visible-xs" href="/"><img src="/static/img/cos-white2.png" class="osf-navbar-logo" width="27" alt="COS logo"/> OSF</a>
+        <div class="navbar-header">
+            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#secondary-navigation" aria-label="Toggle secondary navigation"}}>
+                <span class="sr-only">Toggle navigation</span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+                <span class="icon-bar"></span>
+            </button>
+        <a class="navbar-brand" href="/" aria-label="Go home"><span class="osf-navbar-logo"></span></a>
+        <div class="service-name">
+            <a data-bind="attr: {href: osfServices[currentService].href}">
+                <span class="hidden-xs"> OSF </span>
+                <span><strong data-bind="text: osfServices[currentService].name"></strong></span>
```

Comment:
Does this need to be rendered client-side? This results in a flash of content:

![flash](https://user-images.githubusercontent.com/2379650/27595057-9461d0b2-5b29-11e7-829f-304cdc132657.gif)